src/Symfony/Component/Translation/Translator.php

```diff
@@ -292,6 +292,8 @@ protected function getLoaders()
      */
     public function getMessages($locale = null)
     {
+        trigger_error('The '.__METHOD__.' method is deprecated since version 2.8 and will be removed in 3.0. Rely on the TranslatorBagInterface instead', E_USER_DEPRECATED);
```

Comment:
I would also add a note that the method is called `getCatalogue()` in the `TranslatorBagInterface`.